README.md

```diff
@@ -1,8 +1,25 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+<p align="center">
+  <a href="https://www.ruby-lang.org/en/">
+    <img src="https://img.shields.io/badge/React-v16.3.1-brightgreen.svg" alt="ruby version">
+  </a>
+</p>
 
-## Available Scripts
+## React-Calculator
+This is a calculator built with the React Javascrupt framework.
 
-In the project directory, you can run:
+### `Built with`
+* React
+* Windows
+* Love and Passion💜 for Code
```

Comment:
**Kindly do this 👇 before merging**

😄 I know you're excited starting React and emotions set in, Awesome job getting this far.
But let be professional, ok? 
Kindly remove `Love and Passion💜 for Code` and replace it with the tool used/to be used

Check out this `README` [template](https://github.com/microverseinc/readme-template) from Microverse.

Reply:
Alright, will do so thanks.